src/component/BooksRender.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RemoveBooks from './Removebook';
+
+const Books = ({ id, title, author }) => (
+  <div className="bookSite">
+    <div className="info">
+      <div className="left-section">
+        <ul>
+          <li className="genre books">Action</li>
+          <div>
+            <li className="booktitle books">{title}</li>
+          </div>
+          <li className="bookauthor books">{author}</li>
+        </ul>
+
+        <div className="book-actions">
+          <button className="book-btn" type="button">
+            Comments
+          </button>
+          <span>|</span>
+          <RemoveBooks id={id} />
+          <span>|</span>
+          <button className="book-btn" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+
+      <div className="center-section">
+        <progress value="60" max="100">
+          {' '}
+          60
+        </progress>
+        <span>60%</span>
```

Comment:
- [ ] Great job working here but I think you can use another thing for your progress to be the same as the Zepling design as required so check this article you can install this React library and you can use the circle progress bar [article](https://www.npmjs.com/package/react-circular-progressbar) ☺💯🔥🚀✨


- Your design
![image](https://user-images.githubusercontent.com/58553711/204062696-a5d05d6e-1151-4e2a-b60b-3925171eefa0.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/204062716-c064391a-ac61-4a19-a35a-4c108661ddc1.png)

Reply:
ok I used React Circular Progressbar